src/features/people/PersonPage/index.js

```diff
@@ -1,3 +1,179 @@
+import { useEffect } from "react";
+import { useDispatch, useSelector } from "react-redux";
+import { useParams } from "react-router-dom";
+import { imageBaseUrl } from "../../../ApiValue";
+import {
+  GenreContainer,
+  GenreItem,
+} from "../../../common/tiles/MovieTile/Genre/styled";
+import {
+  Container,
+  Details,
+  InfoContent,
+  Poster,
+  Title,
+  Year,
+} from "../../../common/tiles/MovieTile/styled";
+import { Wrapper } from "../../movies/MovieList/styled";
+import noPoster from "../../../common/tiles/MovieTile/noPoster.png";
+import {
+  Rate,
+  RatingsContainer,
+  StyledStar,
+  Votes,
+} from "../../../common/tiles/MovieTile/Ratings/styled";
+import {
+  fetchPersonDetailsPage,
+  selectPerson,
+} from "../personPageDetailsSlice";
+import { fetchPersonPage, selectCast, selectCrew } from "./personPageSlice";
+import {
+  TileContainer,
+  MainContainer,
+  Article,
+  Image,
+  SubHeaderInformation,
+  SubHeaderContainer,
+  BigTileTitle,
+  MaxRating,
+} from "../../../common/tiles/BigTile/styled";
+import { SubHeaderDate, SubHeaderPerson, CastCrewHeader } from "./styled";
+
 export const PersonPage = () => {
-  return <div>strona Person</div>;
+  const dispatch = useDispatch();
+
+  const { id } = useParams();
+
+  useEffect(() => {
+    dispatch(fetchPersonPage(id));
+    dispatch(fetchPersonDetailsPage(id));
+  }, [dispatch, id]);
+
+  const cast = useSelector(selectCast);
+  const crew = useSelector(selectCrew);
+  const person = useSelector(selectPerson);
+
+  return (
+    <>
+      {person.map(
+        ({ profile_path, name, birthday, place_of_birth, biography }) => (
+          <MainContainer>
+            <Image src={`${imageBaseUrl}/w342${profile_path}`} />
```

Comment:
tutaj warto już dodać alt = `image of {name}` , albo coś podobnego, bo mamy już konkretne dane, także jak wywali fotke, to żeby było wiadomo co tutaj miało być ;)

Reply:
Pewka, sluszne uwagi z tymi altami